spec/mailers/approve_request_mailer_spec.rb

```diff
@@ -118,7 +118,7 @@
       expect(mail.text_part.body.to_s).to eq("The following request was submitted on #{today_formatted}.  It has been Approved by Jane Smith on #{today_formatted}.\n" \
```

Comment:
```suggestion
      expect(mail.text_part.body.to_s).to include("The following request was submitted on #{today_formatted}.  It has been Approved by Jane Smith on #{today_formatted}.\n" \
```
This test's expected text ends with two newlines `\n\n` on line 121.  But the actual string provided ends with `\n\n\n`.  Changing this test from `eq` to `include` will be more flexible, passing whether there are 2 or 3 newline characters at the end.

Lines 82 and 191 would also need this fix.

Alternatively, you could add an extra `\n` to lines 88, 126, and 199.